exporter/awsemfexporter/metric_translator_test.go

```diff
@@ -792,7 +792,45 @@ func TestTranslateOtToCWMetricWithNameSpace(t *testing.T) {
 	assert.Equal(t, "myServiceNS", met.Measurements[0].Namespace)
 }
 
-func TestTranslateOtToCWMetricWithFiltering(t *testing.T) {
+func TestTranslateUnit(t *testing.T) {
+	metric := pdata.NewMetric()
+	metric.SetName("writeIfNotExist")
+
+	translator := &metricTranslator{
+		metricDescriptor: map[string]MetricDescriptor{
+			"writeIfNotExist": {
+				metricName: "writeIfNotExist",
+				unit:       "Count",
+				overwrite:  false,
+			},
+			"forceOverwrite": {
+				metricName: "forceOverwrite",
+				unit:       "Count",
+				overwrite:  true,
+			},
+		},
+	}
+
+	translateUnitCases := map[string]string{
+		"Count": "Count",
+		"ms":    "Milliseconds",
+		"s":     "Seconds",
+		"us":    "Microseconds",
+		"By":    "Bytes",
+		"Bi":    "Bits",
+	}
+	for input, output := range translateUnitCases {
+		metric.SetUnit(input)
```

Comment:
It's fine here, bu in general, it's better to create a sub-test, like:

```go
t.Run(input, func(tt *testing.T) {
 		metric.SetUnit(input)

		v := translator.translateUnit(&metric)
		assert.Equal(t, output, v)
})
```

The output in case of failures is way better this way.

Reply:
Done. Thanks for your suggestion!